pkg/cloudprovider/provider/vsphere/client.go

```diff
@@ -85,3 +83,41 @@ func (s *Session) Logout() {
 		utilruntime.HandleError(fmt.Errorf("vsphere client failed to logout: %s", err))
 	}
 }
+
+func NewRESTSession(ctx context.Context, config *Config) (*RESTSession, error) {
```

Comment:
So it is not possible to create and manage tags using the current client that we have? I thought they have refactored this and you can use the soap client directly without the rest client.

Reply:
All examples and tag occurrences were with the rest-client. I couldn't find anything with soap client